packages/perseus/src/widgets/__testdata__/expression_testdata.js

```diff
@@ -1,19 +1,14 @@
 // @flow
 import type {
-    PerseusRenderer,
     PerseusExpressionWidgetOptions,
     Version,
+    PerseusItem,
 } from "../../perseus-types.js";
 
-type Item = {
-    question: PerseusRenderer,
-    ...
-};
-
 const createItemJson = (
     widgetOptions: PerseusExpressionWidgetOptions,
     version: Version,
-): Item => {
+): PerseusItem => {
```

Comment:
Using the real PerseusItem type instead of the fake Item. That required adding a few fields to the object.